rancher-config/docker-compose.yml

```diff
@@ -4,11 +4,23 @@ services:
   elasticsearch-updater-profiles:
     image: nhsuk/elasticsearch-updater:${DOCKER_IMAGE_TAG}
     environment:
-      NODE_ENV: ${NODE_ENV}
+      NODE_ENV: production
       JSON_FILE_URL: ${JSON_FILE_URL_PROFILES}
       ES_HOST: ${ES_HOST_PROFILES}
       ES_INDEX: profiles
       CHANGE_THRESHOLD: ${CHANGE_THRESHOLD_PROFILES}
       UPDATE_SCHEDULE: ${UPDATE_SCHEDULE_PROFILES}
     labels:
       io.rancher.container.pull_image: always
+
+  elasticsearch-updater-pharmacies:
+    image: nhsuk/elasticsearch-updater:${DOCKER_IMAGE_TAG}
+    environment:
+      NODE_ENV: production
+      JSON_FILE_URL: ${JSON_FILE_URL_PHARMACIES}
```

Comment:
These new env vars need adding to the `README`. Can the incorrectly named ones for profiles also be updated? I didn't pick up on them last time they were changed e.g. `ES_HOST` => `ES_HOST_PROFILES`, etc.

Reply:
the env vars with the suffices are specific to the rancher configuration files.
These are read from the vault then mapped through to suffix-less ones in the docker-compose file, i.e. `JSON_FILE_URL: ${JSON_FILE_URL_PHARMACIES}`

The env vars in the README refer to the ones exposed by the docker container itself, I'm not sure where we would document the vault/rancher mapping when they differ from the docker variable names.

Reply:
So they are, my mistake.